config/esw.yaml

```diff
@@ -292,4 +292,4 @@ brushless:
         max_torque: 0.13
         multiplier: -1
 
-network_iface: "enp44s0"
+network_iface: "enp0s31f6"
```

Comment:
You should be able to do something like this in network_monitor.py

`import os
iface_list = os.list_dir('/sys/class/net/')`
And then check for an interface that starts with enp